extensions/notebook/src/jupyter/jupyterNotebookManager.ts

```diff
@@ -47,10 +47,10 @@ export class JupyterNotebookManager implements nb.NotebookManager, vscode.Dispos
 
 	dispose() {
 		if (this._sessionManager) {
-			this._sessionManager.shutdownAll().then(() => this._sessionManager.dispose());
+			this._sessionManager.shutdown().then(() => this._sessionManager.dispose());
```

Comment:
what is the problem the shutdownAll could cause? We haven't seen any issue so far related to this, right?

Reply:
The follow on call is disposing the manager anyhow. The expected behavior here is that disposing the manager (which owns the session manager) will fully dispose all sessions.